docs/adr/0002-resource-definition.md

```diff
@@ -34,9 +33,11 @@ class Users
     public float $averageRate;
 }
 
-#[Get]
-#[Put]
-#[Delete]
+#[Resource("/companies/{companyId}/users/{id}", normalization_context=["groups"= [....]]), operations={}]
+#[Resource(normalization_context=["groups"= [....]], operations=[
+ #[Get],
+ #[Post],
+])]
```

Comment:
nested attributes are not a thing sadly :(

Reply:
https://wiki.php.net/rfc/new_in_initializers :) will be a thing

Reply:
then it should be:

```php
#[Resource(normalization_context=["groups"= [....]], operations=[
 new Get(),
 new Post(),
])]
```

😛